slink-main/activation.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const sr = require('./smartrecruiters');
+const sapActivateEmployee = require('./sap/activateemployee');
+const util = require('./util');
+
+const process = async () => {
+  const applicants = await sr.getApplicantsOnboarding();
+  console.info(`Activation: Collected ${applicants.length} applicants from SmartRecruiters`);
+
+  const splitByFte = util.split(applicant => applicant.fullTime === true);
+  const ftes = splitByFte(applicants);
+
+  console.info(`Non-FTE applicants skipped: ${ftes.rejects.length}`);
+
+  const splitByEmployeeId = util.split(applicant => applicant.employeeId !== null);
+  const ftesWithId = splitByEmployeeId(applicants);
+
+  console.info(`Employees with no employee ID skipped: ${ftesWithId.rejects.length}`);
+
+  const applicantsActivatedInSap =
+    await Promise.all(ftesWithId.matches
+      .map(async (applicant) => {
+        const sapStatus = await sapActivateEmployee.execute(applicant);
```

Comment:
I think here, or some related place, we're going to need to track the fact that the employee was activated.  I did put an `ApplicantTable` in the `template.yml`, and I guess that's the appropriate place to tack.  I honestly have no idea if my key design for that table is appropriate, however.  This is my first Dynamo use in anger.

I was thinking the ApplicantTable could become a cross-reference, keyed by CandidateID, with attributes of Employee ID, stupid Resume Number, and perhaps an array of `{state, date}`.  This last bit is an idle thought.  Interested in _your_ thoughts.

We can put this persistence stuff in a separate card if it seems best.  But basically, we'll need some way to track whether we've activated an employee/applicant so we don't re-do it.